packages/legend-studio/src/stores/editor-state/FileGenerationViewerState.ts

```diff
@@ -26,7 +26,9 @@ export const getTextContent = (
 ): string => {
   switch (format?.toLowerCase()) {
     case EDITOR_LANGUAGE.JSON:
-      return JSON.stringify(JSON.parse(content), undefined, TAB_SIZE);
+      return content.length
```

Comment:
Let's not do it this way. We don't want any crash. So

```suggestion
      return returnUndefinedOnError(() => JSON.stringify(JSON.parse(content), undefined, TAB_SIZE)) ?? content;
```